addOns/pscanrules/src/test/java/org/zaproxy/zap/extension/pscanrules/HashDisclosureScanRuleUnitTest.java

```diff
@@ -103,7 +103,7 @@ void shouldRaiseAlertWhenResponseContainsCookieWithMd5Hash() throws Exception {
     }
 
     @Test
-    void shouldRaiseAlertWhenResponseContainsOsxSha1AtLowThreshold() throws Exception {
+    void shouldRaiseAlertWhenResponseContainsSaltedSha1AtLowThreshold() throws Exception {
```

Comment:
This is a specific JS test, the hash type is irrelevant (should be named accordingly). `shouldNotRaiseAlertWhenResponseContainsOsxSha1InJsAtNonLowThreshold` should be updated as well.